yarn-project/sequencer-client/src/publisher/l1-publisher.test.ts

```diff
@@ -23,14 +26,27 @@ describe('L1Publisher', () => {
 
     header = l2Block.header.toBuffer();
     archive = l2Block.archive.root.toBuffer();
+    txsHash = l2Block.getCalldataHash();
     body = l2Block.bodyToBuffer();
     proof = Buffer.alloc(0);
 
     txSender = mock<L1PublisherTxSender>();
-    txHash = `0x${Buffer.from('txHash').toString('hex')}`; // random tx hash
-    txReceipt = { transactionHash: txHash, status: true } as MinimalTransactionReceipt;
-    txSender.sendProcessTx.mockResolvedValueOnce(txHash);
-    txSender.getTransactionReceipt.mockResolvedValueOnce(txReceipt);
+
+    publishTxHash = `0x${Buffer.from('txHashPublish').toString('hex')}`; // random tx hash
+    processTxHash = `0x${Buffer.from('txHashProcess').toString('hex')}`; // random tx hash
+    publishTxReceipt = {
+      transactionHash: publishTxHash,
+      status: true,
+      logs: [{ data: txsHash.toString('hex') }],
+    } as MinimalTransactionReceipt;
+    processTxReceipt = {
+      transactionHash: processTxHash,
+      status: true,
+      logs: [{ data: '' }],
+    } as MinimalTransactionReceipt;
+    txSender.sendPublishTx.mockResolvedValueOnce(processTxHash);
```

Comment:
Should this not be the `publishTxHash`?

Reply:
Good catch. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4118/commits/19b1eeb879754943d589f4f0174999cf4012c47a